src/components/organisms/configuration/triggers/formParts/scheduler.tsx

```diff
@@ -25,17 +26,35 @@ export const SchedulerFields = () => {
 	}, [cronValue, setValue]);
 
 	return (
-		<div className="relative">
-			<Input
-				aria-label={t("placeholders.cron")}
-				{...register("cron")}
-				isError={!!errors.cron}
-				isRequired
-				label={t("placeholders.cron")}
-				value={cronValue}
-			/>
-
-			<ErrorMessage>{errors.cron?.message as string}</ErrorMessage>
+		<div className="flex flex-col gap-6">
```

Comment:
Add missing E2E tests in https://github.com/autokitteh/web-platform//blob/a46b7b50f4410bfaa92fc31ff548c7f9b104083f/e2e/project/triggers